src/libraries/System.Private.CoreLib/src/System/Collections/BitArray.cs

```diff
@@ -11,35 +11,38 @@
 
 namespace System.Collections
 {
-    // A vector of bits.  Use this to store bits efficiently, without having to do bit
-    // shifting yourself.
+    /// <summary>
+    /// Manages a compact array of bit values, which are represented as <see cref="bool"/>, where
+    /// <see langword="true"/> indicates that the bit is on (1) and <see langword="false"/> indicates
+    /// the bit is off (0).
+    /// </summary>
     [Serializable]
-    [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
+    [TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public sealed class BitArray : ICollection, ICloneable
     {
-        private int[] m_array; // Do not rename (binary serialization)
+        /// <summary>Bit array. Always little endian, even on big endian platforms.</summary>
+        internal int[] m_array; // Do not rename (binary serialization)
```

Comment:
> Do not rename (binary serialization)

Just curious, is this still something we care about?
(I take it this change breaks existing payloads for BE systems.)

Reply:
@GrabYourPitchforks, @bartonjs, have we documented a position on this?